superset-frontend/src/SqlLab/components/SaveQuery/index.tsx

```diff
@@ -166,6 +167,7 @@ export default function SaveQuery({
     </Form>
   );
 
+  console.log('FINDME SaveQuery', query, queryPayload());
```

Comment:
remove this

Reply:
This was just debugging, it wasn't ready for review yet. I pushed up a "clean up" commit [`here`](https://github.com/apache/superset/pull/20975/commits/9f348bddec74224a808ac5ba88c7883e42f78c6a).